feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/presentation/tab/BrowserTabsAdapter.kt

```diff
@@ -60,9 +59,7 @@ class BrowserTabViewHolder(
     fun bind(item: BrowserTabRvItem) = with(itemView) {
         browserTabCard.setOnClickListener { itemHandler.tabClicked(item, browserTabScreenshot) }
         browserTabClose.setOnClickListener { itemHandler.tabCloseClicked(item) }
-        browserTabScreenshot.load(item.tabScreenshotPath?.asFile(), imageLoader) {
-            transformations(TopCropTransformation())
```

Comment:
Can you elaborate why `TopCropTransform` cannot be used? Does it work bad during animation?

Reply:
Yes, the tab bounds and browser bounds may be different and it would be great to see how image expands in animation. It's not possible if we cut this image